pkg/multus/multus.go

```diff
@@ -75,7 +77,7 @@ func saveScratchNetConf(containerID, dataDir string, netconf []byte) error {
 
 	path := filepath.Join(dataDir, containerID)
 
-	err := ioutil.WriteFile(path, netconf, 0600)
+	err := os.WriteFile(path, netconf, 0600)
```

Comment:
does it matter ?

Reply:
I have bump Golang version to 1.22, and the `ioutil` package has been marked deprecated state,